src/pages/BankHomePage.js

```diff
@@ -13,4 +14,18 @@ export class BankHomePage {
   async clickCustomerLoginButton() {
     await this.customerLoginButton.click();
   }
+
+  async clickBankManagerLoginButton() {
+    await this.bankManagerLoginButton.click();
+  }
+
+
+  async assertCustomerLoginButtonIsVisible() {
+    await expect(this.customerLoginButton).toBeVisible();
+  }
+
+  async assertBankManagerLoginButtonnIsVisible() {
```

Comment:
There is a typo in the method name `assertBankManagerLoginButtonnIsVisible`. The word 'Buttonn' should be corrected to 'Button' to maintain consistency and avoid confusion.